tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -932,7 +932,8 @@ var _ = ginkgo.Describe("TiDBCluster", func() {
 		})
 	})
 
-	ginkgo.Context("[Feature: AutoScaling]", func() {
+	// add [Stability] to prevent running auto scaling related cases for now
+	ginkgo.Context("[Stability][Feature: AutoScaling]", func() {
```

Comment:
Suggest moving to the stability test files.

Reply:
fixed in next commit